app/controllers/component-tree.js

```diff
@@ -298,19 +301,19 @@ export default Controller.extend({
     /**
      * Scrolls the main page to put the selected element into view
      */
-    scrollToElement(elementId) {
+    scrollToElement(objectId) {
       this.get('port').send('view:scrollToElement', {
-        elementId
+        elementId: objectId // TODO: what?
       });
     },
 
-    inspectElement(item) {
-      let elementId;
-      let objectId = item.get('view.objectId');
+    /**
+     * View DOM node in browser Elements panel
+     */
+    viewInElementsPanel(item) {
+      const objectId = item.get('view.objectId');
```

Comment:
@nummi did you talk to @rwjblue about what the difference is between `objectId` and `elementId`? It would be good to figure that out, and if we only need one or the other, remove one.